docs/resources/control-flow/functions/action-triggers.md

```diff
@@ -0,0 +1,291 @@
+- **onTertiaryLongPress**: Triggered when the user presses and holds with a tertiary pointer (e.g., three-finger press).
+- **onTertiaryLongPressCancel**: Triggered when a tertiary long press gesture is recognized but does not complete.
+- **onTertiaryLongPressDown**: Triggered when the user first touches the screen with a tertiary pointer intending to long press.
+- **onTertiaryLongPressEnd**: Triggered when the user releases a tertiary long press.
+- **onTertiaryLongPressMoveUpdate**: Triggered as the user moves a tertiary pointer while holding down during a long press.
+- **onTertiaryLongPressStart**: Triggered when the tertiary long press gesture starts after holding down for the required duration.
+- **onTertiaryLongPressUp**: Triggered when the user releases a tertiary long press after the hold duration.
+- **onTertiaryTapCancel**: Triggered when a tertiary tap gesture is recognized but does not complete successfully.
+- **onTertiaryTapDown**: Triggered when the user first touches the screen with a tertiary pointer intending to tap.
+- **onTertiaryTapUp**: Triggered when the user releases the screen after a tertiary tap.
+- **onVerticalDragCancel**: Triggered when a vertical drag gesture is interrupted or canceled.
+- **onVerticalDragDown**: Triggered when the user first touches the screen and initiates a vertical drag.
+- **onVerticalDragEnd**: Triggered when the user ends a vertical drag gesture.
+- **onVerticalDragStart**: Triggered when the user begins a vertical drag gesture.
+- **onVerticalDragUpdate**: Triggered continuously as the user drags vertically.
+
+### Accessing Gesture Detector XY Data
+
+Gesture detectors not only recognize types of gestures but also provide the exact location (XY coordinates) where they occur. This feature is essential for apps that require precise interaction, like drawing applications or interactive maps.
+
+Examples of using XY data include:
+
+- **Drawing Apps:** Use the touch coordinates to let users paint or draw directly on their device screen.
+- **Interactive Maps:** Implement gestures to navigate around the map or adjust the zoom level based on where the user touches.
+- **Dynamic Interfaces:** Create effects that react to touch, like animations that start from where the user taps the screen.
+
+You can access the Gesture Detector XY data after adding the relevant gesture detector triggers. Once added, you can retrieve this data via the Set from Variable menu inside the Action Flow Editor. Depending on your specific needs, you can choose from the following options:
+
+- **Global Position X**: The x-coordinate of the pointer relative to the left edge of the screen when the gesture was triggered.
+- **Global Position Y**: The y-coordinate of the pointer relative to the top edge of the screen when the gesture was triggered.
+- **Local Position X**: The x-coordinate of the pointer relative to the left edge of the widget that detected the gesture.
```

Comment:
> widget that detected the gesture.

Isnt it the widget that has the action triggers applied on? 🤔

Reply:
I think both are the same, it only detects if it has triggers applied. but 'action triggers applied on' is more clear, so made a slight change.